docs/quick-ingestion-guides/powerbi/setup.md

```diff
@@ -0,0 +1,86 @@
+
+      <p align="center">
+   <img width="75%" alt="app_registration" src="https://raw.githubusercontent.com/main/static-assets-fork/main/imgs/guides/powerbi/app-registration.png"/>
+      </p>
+
+   f. On `Register an application` window click on `Register`
+
+   g. The Azure portal will open up the `powerbi-app-connector` window as shown below. On this screen note down the `Application (client) ID` and click on `Add a certificate or secret` to generate a secret for the `Application (client) ID`
+
+      <p align="center">
+   <img width="75%" alt="powerbi_app_connector" src="https://raw.githubusercontent.com/main/static-assets-fork/main/imgs/guides/powerbi/powerbi-connector-window.png"/>
+      </p>
+
+   f. On `powerbi-connector-app | Certificates & secrets` window generate the client secret and note down the `Secret` 
+
+   You need to configure `Application (client) ID` and `Secret` in powerbi connector recipe.
+
+2. **Create an Azure AD Security Group:** You need to add the `Azure AD app` into the security group to control resource permissions for the `Azure AD app`. Follow below steps to create an Azure AD Security Group.
+
+   a. Go to `Azure Active Directory`
+
+   b. Navigate to `Groups` and click on `New group`
+
+   c. On `New group` window fill out the `Group type`,&nbsp; `Group name`, &nbsp;`Group description`. &nbsp;`Group type` should be set to `Security` . &nbsp; `New group` window is shown in below screenshot.
+
+      <p align="center">
+   <img width="75%" alt="powerbi_app_connector" src="https://raw.githubusercontent.com/main/static-assets-fork/main/imgs/guides/powerbi/new-group-window.png"/>
+      </p>
+
+   d. On `New group` window click on `No members selected` and add `Azure AD app` i.e. *powerbi-connector-app* as member
+
+   f. On `New group` window click on `Create` to create the security group `powerbi-connector-app-security-group`.
+
+3. **Assign privileges to powerbi-connector-app-security-group:**  You need to add the created security group into PowerBI portal to allow resource access. Follow below steps to allow `powerbi-connector-app-security-group` to access PowerBI resources.
+
+   a. Login to https://app.powerbi.com/
+   
+   b. Go to `Settings` -> `Admin Portal`
+
+   c. On `Admin Portal` navigate to `Tenant settings` as shown in below screenshot.
+
+      <p align="center">
+   <img width="75%" alt="powerbi_admin_portal" src="https://raw.githubusercontent.com/main/static-assets-fork/main/imgs/guides/powerbi/powerbi-admin-portal.png"/>
+      </p>
+
+   d. **Enable PowerBI API:** Under `Tenant settings` -> `Developer settings` -> `Allow service principals to use Power BI APIs` add the previously created security group i.e. *powerbi-connector-app-security-group* into `Specific security groups (Recommended)`
+
+   e. **Enable Admin API Settings:** Under `Tenant settings` -> `Admin API settings` enable below options 
```

Comment:
enable below options -> enable the following options

Reply:
done